src/strategies/spookyswap/examples.json

```
@@ -0,0 +1,33 @@
+[
+  {
+    "name":"Example spookyswap query",
+    "strategy":{
+      "name":"spookyswap",
+      "params":{
+        "boo":{
+          "numerator":3,
+          "denominator":1
+        },
+        "lp": {
+          "numerator":5,
+          "denominator":1
+        },
+        "vaultTokens":[
+          {
+            "symbol":"BOO in beefy",
+            "address":"0xEe3a7c885Fd3cc5358FF583F2DAB3b8bC473316f",
+            "numerator":1,
+            "denominator":1,
+            "decimals":18
+          }
+        ]
+      }
+    },
+    "network":"250",
+    "addresses":[
+      "0x1F0C5a9046f0db0e8b651Cd9E8e23ba4Efe4B86d",
+      "0x95478C4F7D22D1048F46100001c2C69D2BA57380"
+    ],
+    "snapshot":"5500150"
```

Snapshot must be a number
```suggestion
    "snapshot": 5500150
```